Modules/EMCAL/src/PayloadPerEventDDLCheck.cxx

```diff
@@ -0,0 +1,109 @@
+#include "QualityControl/MonitorObject.h"
+#include "EMCAL/PayloadPerEventDDLCheck.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "QualityControl/Quality.h"
+
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TLatex.h>
+#include <TList.h>
+#include <TRobustEstimator.h>
+#include <ROOT/TSeq.hxx>
+#include <iostream>
+#include <vector>
+#include <TCanvas.h>
+#include <string>
+
+using namespace std;
+
+namespace o2::quality_control_modules::emcal
+{
+// configure threshold-based checkers
+auto nChiSqMedPayloadThresh = mCustomParameters.find("ChiSqMedPayloadThresh");
+if (nChiSqMedPayloadThresh != mCustomParameters.end()) {
+  try {
+    mChiSqMedPayloadThresh = std::stod(nChiSqMedPayloadThresh->second);
+  } catch (std::exception& e) {
+    ILOG(Error, Support) << ("Value {} not a double", nChiSqMedPayloadThresh->second.data()) << ENDM;
```

Comment:
Doesn't work, must be
``` c++
ILOG(Error, Support) << "Value " <<  nChiSqMedPayloadThresh->second << " not a double" << ENDM;
```
Did you try to build QualityControl?

In addition, when using c++ streams you can use strings directly, as well as other objects that implement operator<<, and don't need to access the C-string.